core/block_svg.ts

```diff
@@ -1819,4 +1823,28 @@ export class BlockSvg
       );
     }
   }
+
+  /** See IFocusableNode.getFocusableElement. */
+  getFocusableElement(): HTMLElement | SVGElement {
+    return this.pathObject.svgPath;
+  }
+
+  /** See IFocusableNode.getFocusableTree. */
+  getFocusableTree(): IFocusableTree {
+    return this.workspace;
+  }
+
+  /** See IFocusableNode.onNodeFocus. */
+  onNodeFocus(): void {
+    if (!this.isShadow()) {
```

Comment:
What happens if it is a shadow block?

Reply:
I'm also not sure. I think call `setSelected` directly here and do some manual verification, then file a ticket to follow up with comprehensive tests for that case.

Reply:
So far I'm not seeing any issues, so I don't think there's a particular concern to address.

I've made a note in #8915 regarding this specific case.